pages/onboarding_new_cluster.ipynb

```diff
@@ -0,0 +1,826 @@
+    "        clientID: %s\n",
+    "        clientSecret:\n",
+    "          name: operate-first-sso-secret\n",
+    "        extraScopes: []\n",
+    "        issuer: https://keycloak-keycloak.apps.moc-infra.massopen.cloud/auth/realms/operate-first\n",
+    "      type: OpenID\n",
+    "\"\"\" % (CLUSTER_NAME)\n",
+    "%store text_input >cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/cluster_patch.yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'cluster-scope/overlays/prod/moc/my-cluster/oauths/cluster_patch.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "List secret-generator, cluster patch, cluster admins and configuration of SSO in `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/kustomization.yaml`:\n",
+    "```\n",
+    "\n",
+    "```"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "source": [
+    "text_input=\"\"\"\\\n",
+    "resources:\n",
+    "  - ../common\n",
+    "  - ../../../../base/config.openshift.io/oauths/cluster\n",
+    "  - ../../../../base/user.openshift.io/groups/cluster-admins\n",
+    "\n",
+    "generators:\n",
+    "  - secret-generator.yaml\n",
+    "\n",
+    "patchesStrategicMerge:\n",
+    "  - oauths/cluster_patch.yaml\n",
+    "\"\"\"\n",
+    "%store text_input >cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/kustomization.yaml"
+   ],
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": [
+      "Writing 'text_input' (str) to file 'cluster-scope/overlays/prod/moc/my-cluster/kustomization.yaml'.\n"
+     ]
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "#### Finalize\n",
+    "\n",
+    "Please stage your changes and send them as a PR against the [operate-first/apps](https://github.com/operate-first/apps) repository. \n",
+    ":::{note}\n",
+    "Make sure that following files/ have been modified/added:\n",
+    "- [x] `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/groups/cluster-admins.yaml`\n",
+    "- [x] `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/kustomization.yaml`\n",
+    "- [x] `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/operate-first-sso-secret.enc.yaml`\n",
+    "- [x] `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/secret-generator.yaml`\n",
+    "- [x] `cluster-scope/overlays/prod/$CLUSTER_REGION/$CLUSTER_NAME/oauths/cluster_patch.yaml`\n",
+    "- [x] `acm/overlays/$CLUSTER_REGION/infra/klusterletaddonconfigs/$CLUSTER_NAME.yaml`\n",
+    "- [x] `acm/overlays/$CLUSTER_REGION/infra/kustomization.yaml`\n",
+    "- [x] `keycloak/overlays/$CLUSTER_REGION/infra/clients/$CLUSTER_NAME.enc.yaml`\n",
+    "- [x] `keycloak/overlays/$CLUSTER_REGION/infra/secret-generator.yaml`\n",
```

Comment:
`s/$CLUSTER_REGION/moc/`

Infra is located in a single region only.

Reply:
Resolved in latest commit.